about.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/all.css"
+    integrity="sha384-SZXxX4whJ79/gErwcOYf+zWLeJdY/qpuqC4cAa9rOGUstPomtqpuNWT9wdPEn2fk" crossorigin="anonymous">
+  <link rel="stylesheet" href="style.css">
+  <title>About WeSingAfrica</title>
+</head>
+
+<body>
+
+  <!-- Mobile Menu -->
+  <div id="wesing_menu" class="mobile-menu">
+    <ul>
+      <li class="header-link"><a href="index.html">Home</a></li>
+      <li class="header-link"><a href="#">About</a></li>
+      <li class="header-link"><a href="./index.html#project_schedule">Program</a></li>
+      <li class="header-link"><a href="#previous_events">Previous Events</a></li>
+      <li class="header-link"><a href="./index.html#wesing_guests">Guests</a></li>
+    </ul>
+    <img src="./images/Icon.png" alt="Close Menu" class="close" id="close-menu">
+  </div>
+  <!-- End Of Mobile Menu -->
+
+  <header id="main-nav">
+    <!-- First Navigation Bar -->
+    <div class="logout-nav">
+      <nav id="first-nav">
+        <ul>
+          <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+          <li><a href="#"><i class="fab fa-facebook-f"></i></a></li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </nav>
+    </div>
+
+    <!-- Second Navigation Bar -->
+    <div class="hamburger">
+      <i class="fas fa-bars fa-2x" id="bars"></i>
+    </div>
+    <div class="nav-with-image">
+      <div>
+
+        <a href="index.html"><img src="./images/wesing-logo.png" alt="Header Logo"></a>
+      </div>
+
+      <nav>
+          <ul>
+            <li class="header-link"><a href="index.html">Home</a></li>
+            <li class="header-link"><a href="#" class="current">About</a></li>
+            <li class="header-link"><a href="./index.html#project_schedule">Program</a></li>
+            <li class="header-link"><a href="#previous_events">Previous Events</a></li>
+            <li class="header-link"><a href="./index.html#wesing_guests">Guests</a></li>
+          </ul>
+      </nav>
+    </div>
+  </header>
+
+  <main id="about-main">
+    <!-- Headline Section -->
+
+    <section class="about_header_section">
+      <div class="wesing_header">
+        <div class="intro">
+          <h2 class="hello">"Hello! Worshipper"</h2>
```

Comment:
- [ ] It looks like you're not using a `<h1>` tag on your page, please add one. Is important for SEO purposes. 🐦